contracts/lib/xERC4626.sol

```diff
@@ -12,95 +12,95 @@ import {IxERC4626} from "../interfaces/IxERC4626.sol";
+        // add unlocked rewards to stored total
+        uint256 unlockedRewards = (lastRewardAmount_ * (block.timestamp - lastSync_)) / (rewardsCycleEnd_ - lastSync_);
+        return storedTotalAssets_ + unlockedRewards;
+    }
 
-    if (block.timestamp >= rewardsCycleEnd_) {
-      // no rewards or rewards fully unlocked
-      // entire reward amount is available
-      return storedTotalAssets_ + lastRewardAmount_;
+    // Update storedTotalAssets on withdraw/redeem
+    function beforeWithdraw(uint256 amount, uint256 shares) internal virtual override {
+        super.beforeWithdraw(amount, shares);
+        storedTotalAssets -= amount;
     }
 
-    // rewards not fully unlocked
-    // add unlocked rewards to stored total
-    uint256 unlockedRewards = (lastRewardAmount_ * (block.timestamp - lastSync_)) / (rewardsCycleEnd_ - lastSync_);
-    return storedTotalAssets_ + unlockedRewards;
-  }
+    // Update storedTotalAssets on deposit/mint
+    function afterDeposit(uint256 amount, uint256 shares) internal virtual override {
+        storedTotalAssets += amount;
+        super.afterDeposit(amount, shares);
+    }
 
-  // Update storedTotalAssets on withdraw/redeem
-  function beforeWithdraw(uint256 amount, uint256 shares) internal virtual override {
-    super.beforeWithdraw(amount, shares);
-    storedTotalAssets -= amount;
-  }
+    /// @notice Distributes rewards to xERC4626 holders.
+    /// All surplus `asset` balance of the contract over the internal balance becomes queued for the next cycle.
+    function syncRewards() public virtual {
+        uint192 lastRewardAmount_ = lastRewardAmount;
+        uint32 timestamp = block.timestamp.safeCastTo32();
 
-  // Update storedTotalAssets on deposit/mint
-  function afterDeposit(uint256 amount, uint256 shares) internal virtual override {
-    storedTotalAssets += amount;
-    super.afterDeposit(amount, shares);
-  }
+        if (timestamp < rewardsCycleEnd) revert SyncError();
 
-  /// @notice Distributes rewards to xERC4626 holders.
-  /// All surplus `asset` balance of the contract over the internal balance becomes queued for the next cycle.
-  function syncRewards() public virtual {
-    uint192 lastRewardAmount_ = lastRewardAmount;
-    uint32 timestamp = block.timestamp.safeCastTo32();
+        uint256 storedTotalAssets_ = storedTotalAssets;
+        uint256 nextRewards = asset.balanceOf(address(this)) - storedTotalAssets_ - lastRewardAmount_;
 
-    if (timestamp < rewardsCycleEnd) revert SyncError();
+        storedTotalAssets = storedTotalAssets_ + lastRewardAmount_; // SSTORE
 
-    uint256 storedTotalAssets_ = storedTotalAssets;
-    uint256 nextRewards = asset.balanceOf(address(this)) - storedTotalAssets_ - lastRewardAmount_;
+        uint32 end = ((timestamp + rewardsCycleLength) / rewardsCycleLength) * rewardsCycleLength;
 
-    storedTotalAssets = storedTotalAssets_ + lastRewardAmount_; // SSTORE
+        if (end - timestamp < rewardsCycleLength / 20) {
```

Comment:
whats with the changes here? can we reset this file to head? 
ill add any auto lint stuff later

Reply:
don't have any changes on this file. all changes are from prettier such as removing spaces